  @org.junit.Test(expected = ArrayIndexOutOfBoundsException.class) // Noncompliant [[sc=19;ec=66;secondary=12,13]] {{Move assertions into separate method or use assertThrows or try-catch instead.}}
  public void testException2() {
    throwingMethod();
    org.junit.Assert.assertTrue(true);
    org.junit.Assert.assertTrue(true);
  }

